pkgs/development/python-modules/rkm-codes/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  pname = "rkm-codes";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "KenKundert";
+    repo = "rkm_codes";
+    rev = "v${version}";
+    sha256 = "sha256-r4F72iHxH7BoPtgYm1RD6BeSZszKRrpeBQccmT4wzuw=";
+  };
+  buildInputs = [ pytest-runner ];
+
+  # this has a circular dependency on quantiphy
+  preBuild = ''
+    sed -i '/quantiphy/d' ./setup.py
+  '';
+
+  # tests require quantiphy import
+  doCheck = false;
```

Comment:
Please add `pythonImportsCheck`.

Reply:
This is one that was confusing to me - the `rkm-codes` and `quantiphy-eval` packages both are dependencies of `quantiphy` (at least for the tests), but they themselves have `quantiphy` as a dependency. Have you seen other packages like this? If so, how do you handle that within nixpkgs? Attempting to add an imports check here will cause an attempt to import `quantiphy`

Reply:
Circular dependencies are always a pain. There were some workarounds present involving setting `doCheck = false;` in `python-modules.nix` and then inherit `doCheck`. Like for `sanic`:

```nix
  pytest-sanic = callPackage ../development/python-modules/pytest-sanic {
    sanic = self.sanic.override { doCheck = false; };
  };
```

Often there is not much value.